app/commands/v2/unpublish.rb

```diff
@@ -132,7 +132,9 @@ def find_unpublishable_content_item(content_id)
         end
 
         filter = ContentItemFilter.new(scope: ContentItem.where(content_id: content_id))
-        filter.filter(locale: locale, state: allowed_states).first
```

Comment:
I suppose we could have raised if `.count > 1`, but it is valid for there to be more than one item, in this case because it finds all unpublished and published items.